app/services/articles/feed.rb

```diff
@@ -82,6 +83,13 @@ def mix_default_and_more_random
       end
     end
 
+    # Test variation: the more comments a post has, the higher it's rated!
+    def more_comments
```

Comment:
I think it would be nice to add a suffix to the filed test method names, e.g. `more_comments_experiment` or something like that. Otherwise it could be a bit confusing why something called `more_comments` loads articles.

Reply:
Great suggestion!